Dockerfile

```
@@ -1,4 +1,16 @@
-FROM alpine:latest
-RUN apk add --update bash docker jq
+FROM golang:1.11.0-alpine
```

```suggestion
FROM golang:1.11-alpine
```
At least we get the latest of the 1.11 train